packages/jest-validate/src/exampleConfig.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type {ValidationOptions} from './types';
+
+const config: ValidationOptions = {
+  comment: '  A comment',
+  condition: () => false,
```

Comment:
I'm using `pretty-format` by default and it only shows the function constructor. I think we can adjust this in another iteration though...